frontend/src/components/AutoJoinEmailsInput/index.tsx

```diff
@@ -0,0 +1,117 @@
+import { useAuthContext } from '@authentication/AuthContext'
```

Comment:
Most of this code was copied from the `AutoJoinForm` component with some tweaks to get the UI looking the way we want.